app/templates/components/forms/wizard/attendee-step.hbs

```diff
@@ -104,10 +104,12 @@
       {{t 'Forward'}}
       <i class="right chevron icon"></i>
     </button>
-    <button class="blue ui three field right labeled icon button {{if isLoading 'disabled'}}" type="button" {{action 'saveDraft'}}>
-      {{t 'Save draft'}}
-      <i class="save icon"></i>
-    </button>
+    {{#if (not-eq data.event.state 'published')}}
```

Comment:
The button should be visible regardless the event state.

Reply:
I don't think , We need this for an already published event and it was implemented the same way in `Basic Detail Form`.